src/compile/selection/transforms/inputs.ts

```diff
@@ -0,0 +1,55 @@
+import {TransformCompiler} from './';
+import {stringValue} from '../../../util';
+import {warn} from '../../../log';
```

Comment:
@arvind prefers it as well. It allows you to namespace functions and enables autocomplete.